docs/user-guide/01-get-started.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: user-guide
 permalink: /user-guide/get-started
+redirect_to: https://docs.shader-slang.org/en/latest/external/slang/docs/user-guide/01-get-started.html
```

Comment:
Can we use macros here so we don't have to repeat the file name (01-get-started) on every page?